free-1.md

```diff
@@ -57,12 +72,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I would say it is 60-40 just because the graphical interpretation is going to be difficult. The language aspects could be applied to the in-between text language which will move more of the time would go to language aspects.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+It really comes down to whether the in-between text language is used or the graphical language that I envision people using. The text-only language seems feasible in the time we have in class.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This is a good idea for a project because it has built-in project scale control (the implementor can decide later if they want to do the graphical interface).
+This may be a poor project if the implmentor does not think this project is an interesting problem to solve. It very well may be that this problem is too small to require a DSL.
```

Comment:
I think this is an interesting idea, but I'm would worry that it may not be a good DSLs project. As you mentioned, the graphical interface is probably the best way to interact with this language; but that doesn't give you much chance to work on language design. If you were to go forward with this project, I would encourage you to work on the text-based version first, or possibly the intermediate representation. However, I realize that these requirements might make the project less interesting to you.